rethinkdb/ast.py

```diff
@@ -20,13 +20,22 @@
 
 import base64
 import binascii
-import collections
+import sys
+
+if sys.version_info.major * 10 + sys.version_info.minor < 33:
```

Comment:
You can just use sys.version_info to compare like a tuple `sys.version_info < (3, 3)`.

```
python3                                                                     
Python 3.10.0a0 (heads/master:f2e161c279, Aug 10 2020, 13:20:03) 
[GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.version_info < (3, 3)
False
>>> sys.version_info > (3, 3)
True
>>> tuple(sys.version_info)
(3, 10, 0, 'alpha', 0)
```

Reply:
Absolutely right, I do the fix

Reply:
Oh seen... it's fixed in a previous branch. Sorry.

Reply:
I redo the fix. As #187 dates from 12 months and I don't see the fix in the release made on December. I don't know if the 2 to 3 branch is still active.
Thanks.